### ISSUE TITLE
explain fails when using array condition

### ISSUE THREAD

**Issue by Svelix** [rails-sqlserver/activerecord-sqlserver-adapter#673]
When calling explain for a query that involves a where clause including a string and an array, an exception is thrown:

```
User.where(id: [1,2], first_name: "John").explain
=> User Load (1.9ms)  EXEC sp_executesql N'SELECT [users].* FROM [users] WHERE [users].[id] IN (1, 2) AND [users].[first_name] = @0', N'@0 nvarchar(4000)', @0 = N'John'  [["first_name", nil]]
NoMethodError: undefined method `[]' for nil:NilClass
```

The problem is in /lib/active_record/connection_adapters/sqlserver/core_ext/explain.rb#26 were the query is split at the first comma in the middle of the expresion.

**Comment by wpolicarpo** [comment]
@Svelix I've just released `v5.2.0.rc2` with a fix for this issue. Could you test and let me know if that is working for you?

**Comment by Svelix** [comment]
@wpolicarpo Works for me! Thank you!
